tests/java/less_inheritance.java.TODO

```diff
@@ -0,0 +1,19 @@
+// ERROR: match
+public class A {
+  public static foo() {
+    return 1;
+  }
+}
+
+// ERROR: match
+public class B extends A {
```

Comment:
No, this should be class A extends B { ...}
we do not want class A { .. } to match public class B extends A { }

Reply:
ok